dspace/config/spring/rest/exposed-properties-configuration.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd">
+
+    <!-- Define which configuration properties are exposed through the http://<dspace.server.url>/api/config/properties/
+         rest endpoint. If a rest request is made for a property which exists, but isn't listed here, the server will
+         respond that the property wasn't found.-->
+    <util:list id="exposedConfigurationProperties" value-type="java.lang.String">
+        <value>google.analytics.key</value>
```

Comment:
Since this is a simple array/list of property keys, why not make this configurable within the `rest.cfg` file?

In other words, it'd be much easier to set this like:
```
rest.properties.exposed = google.analytics.key
rest.properties.exposed = second.property
```

Then in the `ConfigurationRestRepository`, you can access these values by doing:
```    
@Value("${rest.properties.exposed}")
private String[] exposedProperties;
```

I'd rather this setting be in our normal configuration as that makes it much more flexible. It's easier to override in local.cfg, via environment variable or System property setting.